std/algorithm/sorting.d

```diff
@@ -145,20 +145,20 @@ Checks whether a forward range is sorted according to the comparison
 operation $(D less). Performs $(BIGOH r.length) evaluations of $(D
 less).
 
-Unlike $(LREF isSorted), $(LREF isStrictlyMonotonic) does not allow for equal values,
+Unlike isSorted, isStrictlyMonotonic does not allow for equal values,
```

Comment:
Well anything that's not part of the English prose needs to be formatted differently. This is code so it should come out in code font by whichever means. N.B. "isSorted" is not even in the dictionary.